cmd/security-agent/app/runtime.go

```diff
@@ -253,3 +276,68 @@ func startRuntimeSecurity(hostname string, stopper restart.Stopper, statsdClient
 
 	return agent, nil
 }
+
+func downloadPolicy(cmd *cobra.Command, args []string) error {
+	if !coreconfig.Datadog.IsSet("api_key") {
+		return errors.New("no API key is set")
+	}
+
+	if !coreconfig.Datadog.IsSet("app_key") {
+		return errors.New("no application key is set")
+	}
+
+	apiKey := coreconfig.Datadog.GetString("api_key")
```

Comment:
What about simply checking that apiKey and appKey are not empty ?

Reply:
It would be better yes